lighthouse-core/gather/driver.js

```diff
@@ -1217,6 +1217,41 @@ class Driver {
     return flattenedDocument.nodes ? flattenedDocument.nodes : [];
   }
 
+  /**
```

Comment:
probably worth a comment description for these since the parameters/return values are only primitive types? And maybe a word on what `undefined` indicates

Reply:
done